airbyte-webapp/src/components/EntityTable/components/StatusCell.tsx

```diff
@@ -6,28 +6,37 @@ import styled from "styled-components";
 import { Button } from "components/ui/Button";
 import { Switch } from "components/ui/Switch";
 
+import { SchemaChange } from "core/request/AirbyteClient";
 import { useEnableConnection } from "hooks/services/useConnectionHook";
 
+import { ChangesStatusIcon } from "./ChangesStatusIcon";
+import styles from "./StatusCell.module.scss";
+
 interface IProps {
   allowSync?: boolean;
   enabled?: boolean;
   isSyncing?: boolean;
   isManual?: boolean;
   id: string;
   onSync: (id: string) => void;
+  schemaChange: SchemaChange;
 }
 
 const ProgressMessage = styled.div`
   padding: 7px 0;
 `;
 
-const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, isSyncing, onSync, allowSync }) => {
+export const StatusCell: React.FC<IProps> = ({ enabled, isManual, id, isSyncing, onSync, allowSync, schemaChange }) => {
   const { mutateAsync: enableConnection, isLoading } = useEnableConnection();
-
-  const [{ loading }, OnLaunch] = useAsyncFn(async (event: React.SyntheticEvent) => {
-    event.stopPropagation();
-    onSync(id);
-  }, []);
+  const isSchemaChangesFeatureEnabled = process.env.REACT_APP_AUTO_DETECT_SCHEMA_CHANGES_FEATURE_ENABLED === "true";
```

Comment:
This can be hoisted outside the component since it doesn't change.

Reply:
It's fine to leave it here because this will soon change to use the experiment hook